package.json

```diff
@@ -60,6 +60,7 @@
     "babel-eslint": "^10.1.0",
     "babel-jest": "^29.0.2",
     "babel-loader": "^9.1.0",
+    "conventional-changelog-conventionalcommits": "^7.0.2",
```

Comment:
Do we need this package?

Reply:
i think we do - im reading this [here](https://github.com/semantic-release/release-notes-generator#configuration:~:text=then%20subject)-,Configuration,-Options) and its saying its needed for that